code/src/components/ListMessages.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react'
+import moment from 'moment'
+
+import { HAPPY_THOUGHTS_API } from '../reusable/urls'
+import LikePost from './LikePost'
+
+const ListMessages = ({ messageList, setMessageList }) => {
+
+  const fetchMessageList = () => {
+    fetch(HAPPY_THOUGHTS_API)
+      .then(response => response.json())
+      .then(messages => setMessageList(messages))
+      .catch(err => console.error(err))
+  }
+
+  useEffect(fetchMessageList, [])
+
+  return (
+    <>
+      {messageList.map(post => (
+        <article key={post._id} className="message-container">
```

Comment:
Here I actually think an article tag is a viable option since you’re displaying the content as an independent message. 😀